### PR TITLE
Refactored UI and fixed tests for Events Page

### PR DESCRIPTION
**Pull request by Annabel1054**
**Before:** 
image

**After:** 
image
image

### CODE REVIEW

**Review by aashrav — CHANGES_REQUESTED** [review]
Great Work! Just a couple things I think should be fixed

1.  Events that have already happened(before current date) should not be shown on the Event Page
2. Right now the events are shown in descending order meaning that events that are next month would render above the events that are tomorrow. This should be the other way around so that people can see which events are coming up on the top itself. 
3. This is a small issue of overflowing description when a word is too long. Can't think of when someone would enter such a large word, but defiantly something to consider 
image
4. Another suggestion is making the title a different color so that it stands out more.

File: src/Pages/EventManager/EventManager.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ export default function EventManager(props) {
         <Button className='create-event' onClick={toggleNewEvent}>
           New Event
         </Button>
+        <div> <br></br> </div>
```
Comment (law-nguyen) [inline src/Pages/EventManager/EventManager.js:73]: what is this for?

Comment (Annabel1054) [reply src/Pages/EventManager/EventManager.js:73]: It creates a space between the "new event" button and "click on any event below to edit or delete it"
image

**Review by law-nguyen — CHANGES_REQUESTED** [review]
This is a great start!
1. I would address the changes that aash proposes.
2. when you shrink the page down, maybe changing the card layout to have the event on the bottom and the picture on top of it would be better. Right now the scaling is a bit off :<

image


I would propose adding functionality to swap displays to something like this when the width of the screen gets too small 
image

**Comment by Annabel1054** [comment]
@aashrav the events are now chronologically ordered and the events that have passed will no longer show up on the events page.
image
image

**Comment by Annabel1054** [comment]
@law-nguyen fixed!!
image

File: src/Pages/Events/EventCard.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import React from 'react';
 import { Row } from 'reactstrap';
```
Comment (law-nguyen) [inline src/Pages/Events/EventCard.js:2]: delete this line! `Row` is never used

File: src/Pages/Events/EventList.js [hunk 3]
```diff
@@ -30,6 +32,24 @@ function AnnouncementList() {
...
+        currDate.setDate(currDate.getDate() -1);
+        let validList = [];
+        eventList.forEach(item => {
+          let date = new Date(item.eventDate);
+          if (date >= currDate) {
+            validList.push(item);
+          }
+        }, setValidList(validList), setGetFiltered(false));
+      } catch (error) {
+        const alertText = 'There are no events to filter!';
```
Comment (law-nguyen) [inline src/Pages/Events/EventList.js:48]: This isn't used either in the code, so it's alright to take it out for now.

**Review by law-nguyen — CHANGES_REQUESTED** [review]
Almost done! There are a couple of warnings that pop up when running build that you gotta fix.

image

**Review by IsitaB — CHANGES_REQUESTED** [review]
Screen Shot 2020-11-26 at 11 22 51 AM
fix this

**Comment by qsaml** [comment]
functionality of creating and displaying events works, but running `npm run frontend-test` on my end returns a few errors

**Comment by evanugarte** [comment]
this is a big pr

**Pull request closed without merging**